spec/presenters/activity_presenter_spec.rb

```diff
@@ -270,4 +270,15 @@
       expect(described_class.new(project).link_to_roda).to eq "http://test.local/organisations/#{project.organisation.id}/activities/#{project.id}/details"
     end
   end
+
+  describe "#transaction_total" do
+    it "returns the transaction total as a formatted number with currency symbol" do
+      project = create(:project_activity)
+      transaction_1 = create(:transaction, parent_activity: project)
+      transaction_2 = create(:transaction, parent_activity: project)
+
+      expect(described_class.new(project).transaction_total)
+        .to eq ActionController::Base.helpers.number_to_currency((transaction_1.value + transaction_2.value), unit: "£")
```

Comment:
My preference is to explicitly set `value` in `create(:transaction, ... )` and then the expectation is looking for a specific value (e.g. "£1234.56"). 

That way if we somehow break `number_to_currency` we know about it.